imports/api/settings.js

```diff
@@ -20,7 +20,11 @@ const pattern = {
   maxImageSize: Number,
   maxImageWidth: Number,
   email: String,
-  requireVerified: Boolean
+  requireVerified: Boolean,
```

Comment:
Let's add a key that is "storageType" (or something similar), and set it to AWS.
Then, let's preface the other keys with AWS_ (e.g. AWS_bucket). This way, it's clear that these are keys related to AWS storage.

Reply:
That has been implemented in the next branch I have been working on.